sky/engine/bindings/snapshot_generic.dart

```diff
@@ -0,0 +1,19 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// This file serves as a generic snapshot entry point. It used to generate
+// the blobs necessary to create valid kDartIsolateSnapshotBuffer and
+// kDartVmIsolateSnapshotBuffer symbols for VM initialization. 
+
+import 'dart:async';
+import 'dart:collection';
+import 'dart:convert';
+import 'dart:core';
+import 'dart:developer';
+import 'dart:io';
+import 'dart:isolate';
+import 'dart:math';
+import 'dart:mojo.internal';
+import 'dart:ui';
+import 'dart:typed_data';
```

Comment:
What's wrong with the existing `snapshot.dart` ?  The only difference appears to be `dart:jni`, which seems harmless.  We just don't wire up a resolver for `dart:jni` on platforms that don't have Java.